server/src/test/java/org/elasticsearch/cluster/routing/allocation/RebalanceAfterActiveTests.java

```diff
@@ -75,8 +74,6 @@ public Long getShardSize(ShardRouting shardRouting) {
             assertThat(clusterState.routingTable().index("test").shard(i).size(), equalTo(2));
             assertThat(clusterState.routingTable().index("test").shard(i).shard(0).state(), equalTo(UNASSIGNED));
             assertThat(clusterState.routingTable().index("test").shard(i).shard(1).state(), equalTo(UNASSIGNED));
-            assertThat(clusterState.routingTable().index("test").shard(i).shard(0).currentNodeId(), nullValue());
-            assertThat(clusterState.routingTable().index("test").shard(i).shard(1).currentNodeId(), nullValue());
```

Comment:
Could we keep these assertions?

Reply:
We could, however they do not make much sense, above lines assert that shard is unassigned and according to org.elasticsearch.cluster.routing.ShardRouting#assertConsistent we should not have currentNodeId set.